src/lang/english.txt

```diff
@@ -4134,6 +4131,13 @@ STR_AI_LIST_ACCEPT_TOOLTIP                                      :{BLACK}Select h
 STR_AI_LIST_CANCEL                                              :{BLACK}Cancel
 STR_AI_LIST_CANCEL_TOOLTIP                                      :{BLACK}Don't change the script
 
+STR_SCREENSHOT_CAPTION											:{WHITE}Screenshot
+STR_SCREENSHOT_SCREENSHOT                                       :{BLACK}Screenshot
+STR_SCREENSHOT_ZOOMIN_SCREENSHOT                                :{BLACK}Fully zoomed in screenshot
+STR_SCREENSHOT_DEFAULTZOOM_SCREENSHOT                           :{BLACK}Default zoom screenshot
+STR_SCREENSHOT_WORLD_SCREENSHOT                                 :{BLACK}Whole map screenshot
+STR_SCREENSHOT_HEIGHTMAP_SCREENSHOT								:{BLACK}Heightmap screenshot
```

Comment:
tab indentation here

Reply:
Fixed in https://github.com/OpenTTD/OpenTTD/pull/7550/commits/b7770416b5c683d5cbc4e6b35f3a77a292c9ef75